colabfold/plot.py

```diff
@@ -2,6 +2,55 @@
 
 import numpy as np
 from matplotlib import pyplot as plt
+from matplotlib import collections as mcoll
+import matplotlib
+from string import ascii_uppercase, ascii_lowercase
+
+pymol_color_list = [
```

Comment:
Could you import that list like `from colabfold.colabfold import pymol_color_list`

Reply:
Well, in fact I noticed there was no import from `colabfold.py` directly in `batch.py` or `plot.py`, so I thought there was a reason for that, and I copied in `plot.py` few functions and variables like `kabsch`, `plot_pseudo_3D`, `add_text` and `plot_ticks` from `colabfold.py`.

So if `colabfold.py` is not deprecated, I should import all functions from colabfold.

Let me know what you think of it.

Reply:
In fact I realized that `colafold.py` was imported, so I remove all copied function and replace it by an import of `colabfold.colabfold`